### PR TITLE
[EdgeDB] Create Language queries | refactor service/repo layers appropriately

### PR DESCRIPTION
**Pull request by bryanjnelson**
[Monday Task](https://seed-company-squad.monday.com/boards/5989610236/pulses/5792968862)

### CODE REVIEW

**Comment by bryanjnelson** [comment]
@CarsonF - I tried to do a little refactoring with this commit [fc0d1fd](https://github.com/SeedCompany/cord-api-v3/pull/3118/commits/fc0d1fd8d46397cde3786cbe699c67e2011bfe69).  I know I need help with the Cypher query, but is any of this approach even correct?

File: src/components/project/dto/list-projects.dto.ts [hunk 1]
```diff
@@ -89,9 +89,23 @@ export abstract class ProjectFilters {
   })
   readonly presetInventory?: boolean;
 
+  @Field({
```
Comment (CarsonF) [inline src/components/project/dto/list-projects.dto.ts:92]: These have been intentionally kept internal.
Idiomatic for GQL is `partner { projects }`. This is just the implementation detail.

File: src/components/project/list-filter.query.ts [hunk 2]
```diff
@@ -64,6 +64,11 @@ export const projectListFilter = (input: ProjectListInput) =>
         ]),
       ],
     }),
+    languageId: filter.pathExists((id) => [
+      node('node'),
+      relation('out', '', 'language', ACTIVE),
```
Comment (CarsonF) [inline src/components/project/list-filter.query.ts:69]:
```suggestion
      relation('out', '', 'engagement', ACTIVE),
      node('', 'LanguageEngagement'),
      relation('out', '', 'language', ACTIVE),
```

**Review by CarsonF — COMMENTED** [review]
> @CarsonF - I tried to do a little refactoring with this commit fc0d1fd. I know I need help with the Cypher query, but is any of this approach even correct?

Yep!

File: src/components/language/language.service.ts [hunk 3]
```diff
@@ -313,17 +252,4 @@ export class LanguageService {
       throw new ServerException('Could not remove location from language', e);
     }
   }
-
-  /**
-   * Check if the language has no engagements that have firstScripture=true.
-   */
-  protected async verifyExternalFirstScripture(id: ID) {
```
Comment (CarsonF) [inline src/components/language/language.service.ts:320]: This needs to be restored in some fashion. It's the only failing test.

**Review by CarsonF — APPROVED** [review]
LGTM! Thanks @bryanjnelson

**Pull request merged**